src/assets/sass/protocol/components/_footer.scss

```diff
@@ -12,111 +12,231 @@
     background: $color-black;
     clear: both;
     color: $color-white;
-    line-height: 1.3;
     margin: 0;
-    padding: $padding-xl 0;
+    padding-top: $spacing-lg;
     width: 100%;
 
-    .mzp-c-footer-primary,
-    .mzp-c-footer-legal {
-        a:link,
-        a:visited {
-            color: $color-white;
-            font-weight: normal;
-            text-decoration: none;
-        }
-
-        a:hover,
-        a:focus,
-        a:active {
-            color: $color-white;
-            text-decoration: underline;
-        }
+    a:link,
+    a:visited {
+        color: $color-white;
+        font-weight: normal;
     }
 
-    h5 {
-        @include text-display-xs;
+    a:hover,
+    a:focus,
+    a:active {
         color: $color-white;
-        margin-bottom: $margin-lg;
+    }
+}
 
-        a:link,
-        a:visited {
-            font-weight: bold;
-        }
+
+/* -------------------------------------------------------------------------- */
+// Footer primary section, contains logo and sections with links
+
+.mzp-c-footer-primary {
+    @include text-body-md;
+    @include border-box;
+    @include clearfix;
+    margin: 0 auto $spacing-2xl;
+
+    @media #{$mq-lg} {
+        margin-bottom: $spacing-xl;
     }
+}
 
-    li {
-        list-style-type: none;
-        margin: 0 0 $margin-md;
+/* -------------------------------------------------------------------------- */
+// Footer Mozilla logo
+
+.mzp-c-footer-primary-logo {
+    margin-bottom: $spacing-xl;
+
+    a {
+        background: url('#{$image-path}/logos/mozilla/white.svg') no-repeat;
+        @include background-size(100px, 32px);
+        @include image-replaced;
+        display: inline-block;
+        height: 32px;
+        width: 100px;
+    }
+}
+
+
+/* -------------------------------------------------------------------------- */
+// Footer section containing list and heading
+
+.mzp-c-footer-sections {
+    @include clearfix;
+
+    @media #{$mq-md} {
+        border-bottom: 1px solid $color-gray-70;
+        padding-bottom: $layout-lg;
+    }
+
+    @media #{$mq-lg} {
+        padding-bottom: $layout-md;
+    }
+}
+
+.mzp-c-footer-section {
+    @include clearfix;
+    margin-bottom: $spacing-sm;
+
+    a:link,
+    a:visited {
+        text-decoration: none;
     }
 
-    nav {
-        @include border-box;
-        @include clearfix;
-        margin: 0 auto;
-        min-width: $content-xs;
+    a:hover,
+    a:focus,
+    a:active {
+        text-decoration: underline;
+    }
+
+    @media (min-width: 480px) and (max-width: 1055px) {
```

Comment:
They're based off our Media Queries but I couldn't find tokens for the values outside of media queries. Can I add them? (What makes something a token instead of a SCSS variable?)

Reply:
Interesting, these values should be in our tokens, but I think you may have just discovered a bug: https://github.com/mozilla/protocol-tokens/blob/master/tokens/content.yml#L21-L30

The values for things like `screen-lg` are mapped to `content` values, whereas they should be mapped to the values here https://github.com/mozilla/protocol-tokens/blob/master/tokens/_aliases.yml#L134-L138

Reply:
I filed an issue for this here https://github.com/mozilla/protocol-tokens/issues/27

Reply:
PR to expose correct screen size tokens: https://github.com/mozilla/protocol-tokens/pull/28